tgui/packages/tgui/interfaces/Mecha/UtilityModulesPane.tsx

```diff
@@ -72,3 +76,50 @@ const SnowflakeEjector = (props: {module: MechaUtility}, context) => {
     </LabeledList>
   );
 };
+
+const SnowflakeExtinguisher = (props: {module: MechaUtility}, context) => {
+  const { act, data } = useBackend<OperatorData>(context);
+  return (
+    <Fragment>
```

Comment:
Why is this <Fragment> and not <>? I'm probably unaware of something.

Reply:
I dunno, last time I wrote tgui we used Fragment.